ansible/roles/miniflux/defaults/main.yml

```diff
@@ -321,21 +311,18 @@ miniflux__golang__dependent_packages:
   - name: 'miniflux'
     upstream_type: '{{ miniflux__upstream_type }}'
     apt_packages: [ 'miniflux' ]
-    gpg:
-      - id: '{{ miniflux__upstream_gpg_key }}'
-        url: 'https://apt.miniflux.app/KEY.gpg'
     url:
       - src: '{{ miniflux__upstream_url_mirror + "download/" + miniflux__upstream_version + "/miniflux-" + miniflux__upstream_platform }}'
-        dest: 'releases/{{ miniflux__upstream_platform }}/miniflux/miniflux/" + miniflux__upstream_version + "/miniflux_" + miniflux__upstream_version + "_linux_amd64'
+        dest: '{{ "releases/" + miniflux__upstream_platform + "/miniflux/v2/" + miniflux__upstream_version + "/miniflux_" + miniflux__upstream_version + "_" + miniflux__upstream_platform.replace("-", "_") }}'
     url_binaries:
-      - src: 'releases/{{ miniflux__upstream_platform }}/miniflux/miniflux/" + miniflux__upstream_version + "/miniflux_" + miniflux__upstream_version + "_linux_amd64'
+      - src: '{{ "releases/" + miniflux__upstream_platform + "/miniflux/v2/" + miniflux__upstream_version + "/miniflux_" + miniflux__upstream_version + "_" + miniflux__upstream_platform.replace("-", "_") }}'
         dest: 'miniflux'
         notify: [ 'Restart miniflux' ]
     git:
       - repo: '{{ miniflux__upstream_git_repository }}'
         version: '{{ miniflux__upstream_version }}'
         build_script: |
-          make clean linux-amd64
+          'make clean {{ miniflux__upstream_platform }}'
```

Comment:
I wanted to use `miniflux__upstream_platform` variable to use the same variable in both ARMv7 (Raspberry Pi 2 Model B) and x86_64 architectures. I didn't try it on other architectures.